application/controllers/logic.py

```diff
@@ -236,9 +236,14 @@ def process_card(card, player, is_discarded, for_wonder):
 def db_add(*args, **kwargs):
     try:
         for value in args:
+            print('args')
             db.session.add_all(value)
```

Comment:
Are we sure that args won't have objects instead of lists passed to it too? Would it be better to have the list guard check in this branch as well?

Reply:
Good question. I am very unsure. What do you think?